tests/main/default-tracks/task.yaml

```
@@ -5,11 +5,12 @@ environment:
     A_TRACK: default
 
 prepare: |
-    snap install http jq
+    snap install jq
+    snap install test-snapd-curl --edge
 
 execute: |
     # first, sanity check that the snap has a default track
-    snap run http GET "https://api.snapcraft.io/v2/snaps/info/$A_SNAP" Snap-Device-Series:16 > info
+    snap run test-snapd-curl.curl -H "Snap-Device-Series:16" "https://api.snapcraft.io/v2/snaps/info/$A_SNAP" > info
```

```suggestion
    snap run test-snapd-curl.curl -H "Snap-Device-Series: 16" "https://api.snapcraft.io/v2/snaps/info/$A_SNAP" > info
```
I don't think it's broken as-is, but I think as per the HTTP spec there is supposed to be some whitespace between the colon and the header value

---

done